src/librustc_metadata/lib.rs

```diff
@@ -7,7 +7,7 @@
 #![feature(nll)]
 #![feature(or_patterns)]
 #![feature(proc_macro_internals)]
-#![feature(specialization)]
+#![feature(specialization)] // FIXME: min_specialization ICEs
```

Comment:
@matthewjasper ICEs shouldn't happen, should they?
```
error: internal compiler error: src/librustc_trait_selection/traits/specialize/mod.rs:101: When translating substitutions for specialization, the expected specialization failed to hold

thread 'rustc' panicked at 'Box<Any>', src/librustc_errors/lib.rs:904:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Reply:
They shouldn't, but the feature has pretty limited testing.